src/main/java/com/sda/groupa/shippingcostcalculator/fuelController/FuelController.java

```diff
@@ -0,0 +1,57 @@
+package com.sda.groupa.shippingcostcalculator.fuelController;
+
+import com.sda.groupa.shippingcostcalculator.fuelModel.Fuel;
+import com.sda.groupa.shippingcostcalculator.fuelService.FuelService;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+
+@Controller
+public class FuelController {
+
+    private final FuelService fuelService;
+
+    public FuelController (FuelService fuelService){
+        this.fuelService=fuelService;
+    }
+
+
+//    public ModelAndView getFuelPageWithListOfFuelings(@PathVariable Long id){
+    @GetMapping(value = "/fuel")
```

Comment:
Pobieramy liste fuelings, wiec fajnie jakby url nam tez to sugerowal: `@GetMapping(value = "/fuelings")` :)